src/ui/components/Library/Team/View/TestRuns/Overview/TestResultListItem.tsx

```diff
@@ -30,7 +30,7 @@ function ViewReplay({ recordingId, passed }: { recordingId: string; passed: bool
 
 function Title({ recording }: { recording: Recording }) {
   const { teamId } = useContext(TeamContext);
-  const path = `${JSON.stringify(recording.metadata.test!.path)}`;
+  const path = `${JSON.stringify(recording.metadata?.test?.path)}`;
```

Comment:
I've added a guard so that it will show the title without a link if the path is undefined (because the path is necessary for the link URL). Not sure if this can happen so let's play it safe.